tests/sentry/grouping/__init__.py

```diff
@@ -191,7 +191,7 @@ def create_event(self):
 )
 
 
-def with_fingerprint_input(name):
+def with_fingerprint_input(name: str) -> Callable[..., None]:
```

Comment:
We already have the more specific type from `with_grouping_inputs` earlier on in the file.

```suggestion
def with_fingerprint_input(name: str) -> pytest.MarkDecorator:
```